realEstate.py

```diff
@@ -92,8 +92,7 @@ def sortData(self):
 		['9844636', '$630,100', '$','153 Black Bear','FOR_SALE','ForSale','TRUE','NULL','Blaine','FALSE'],
 		['7468349', '$110,000', '$','1792 12th Street','FOR_SALE','Null','NULL','FALSE','Biloxi','68500'],
 		['128749', '$830,500', '$','2513 Swan LN','FOR_SALE','Null','NULL','TRUE','Carson City','342400'],
-		['6486438', '$500,900', '$','1104 Old Swanzey','FOR_SALE','ForSale','TRUE','FALSE','Union City','FALSE'],
-		['5348347', '$130,000', '$','1153 Oceanfront','FOR_SALE','Null','NULL','FALSE','Missoula','FALSE']]
+]
```

Comment:
Move the csv file to it's own file. 
Read about comparators to expound the sort such that it is based on the fields